src/components/map/MapComponent.vue

```diff
@@ -55,6 +67,59 @@ onUnmounted(() => {
   vcsApp.destroy()
 })
 
+function removeFilterOnLayers(featureKey: string) {
```

Comment:
Is there a reason why we use `featureKey`? It seems it should be a `layerName` here.